### PR TITLE
Flake8v3

### PR DESCRIPTION
**Pull request by cadenmyers13**
This branch passes all tests

### CODE REVIEW

**Comment by cadenmyers13** [comment]
This branch passes all tests.

File: src/diffpy/pdfgui/gui/temperatureseriespanel.py [hunk 1]
```diff
@@ -288,18 +300,26 @@ def onDelete(self, event):  # wxGlade: TemperatureSeriesPanel.<event_handler>
         self.fillList()
         return
 
+    def create_filename_order(self):
```
Comment (sbillinge) [inline src/diffpy/pdfgui/gui/temperatureseriespanel.py:303]: Why are you wrapping this in a function?  It doesn't seem like it needs to be.

File: src/diffpy/pdfgui/gui/temperatureseriespanel.py [hunk 2]
```diff
@@ -288,18 +300,26 @@ def onDelete(self, event):  # wxGlade: TemperatureSeriesPanel.<event_handler>
...
+        filenames = [f for t, f in self.datasets]
+        numericStringSort(filenames)
+        self.filename_order = dict(zip(filenames, range(len(filenames))))
+        return
+
+    def filename_sortkey(self, tf):
```
Comment (sbillinge) [inline src/diffpy/pdfgui/gui/temperatureseriespanel.py:309]: when we do turn lambdas into functions, mostly they should be private functions (just put an underscore at the beginning of the name).

**Review by sbillinge — COMMENTED** [review]
please see the inline comment.  Most of these lamdas -> functions should be private actually, if you could quickly refactor them all.

**Comment by sbillinge** [comment]
closing. Fixed by #141 and #142

**Pull request closed without merging**